docs/sql-programming-guide.md

```diff
@@ -1405,16 +1405,6 @@ Configuration of Parquet can be done using the `setConf` method on `SQLContext`
     </p>
   </td>
 </tr>
-<tr>
```

Comment:
Please explain your change. Nothing in the PR or JIRA describes why. "Fix SQL Programming Guide" is not a sufficient title or description, so please make more descriptive titles in the future.